tests/integration/integration_test.py

```diff
@@ -1254,6 +1254,16 @@ def print_usage(exitcode, reason=None):
     if 'avro-https' in modes:
         print('=' * 30, 'Verifying AVRO with HTTPS', '=' * 30)
         verify_avro_https(testconf.get('avro-https', None))
+        conf = testconf.get("avro-https-key-with-password", None)
```

Comment:
2 confs are being used in this function. use `key-with-password-conf`.

Reply:
just change the name of conf right ?

Reply:
Thats Done!